### DISCUSSION TITLE
jwt and hls

### DISCUSSION THREAD

**Discussion by rfscheidt** [bluenviron/mediamtx#3169] [Questions]
### Question

Hello everyone,

I enabled the JWT settings in my project and configured a Keycloak.
  mediamtx:
    image: bluenviron/mediamtx
    environment:
       - MTX_AUTHJWTJWKS=http://172.30.5.2:8080/realms/mediamtx/protocol/openid-connect/certs
       - MTX_AUTHMETHOD=jwt
       
For the RTSP protocol, it worked fine by passing the JWT in the URL: rtsp://localhost:8554/test2?jwt=XXX

However, I can't get it to work with the API and HLS.
How should I pass the token in these cases?
I am user 1.6.0 version.

Thanks,

**Comment by aler9** [comment]
The API issue has been fixed in v1.8.0.
The HLS issue is still open and is tracked here: #3275

[a comment by an automated account is not shown]
